src/domain/events/eventSearchForm/EventSearchForm.tsx

```diff
@@ -22,6 +22,8 @@ export type EventSearchFormValues = {
   endDate: Date | null;
   places: string[];
   organisation?: string;
+  organization?: string;
+  organisationId?: string;
```

Comment:
Are you sure this is correct? There is now organization and organisation with z and s?

Reply:
yes, we have a mess with those s and z all over the project. in query string we use z sometimes, schemas have s. and mappers have different values. we basically need to refactor then all places (and backend as well) to make it conventional

Reply:
@Sartonon you left the call when i explained why its a "tricky" solution. so the code is not nice i agree.. but might be temporary working solution.